### PR TITLE
Bug 1121274 - Intermittent uninstallation_test.js | Show Keyboard App after

### PR DESCRIPTION
**Pull request by RudyLu**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dbb06442aadf)

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9a6eca799d27)

File: apps/keyboard/test/marionette/uninstallation_test.js [hunk 1]
```diff
@@ -56,10 +56,16 @@ marionette('Show Keyboard App after uninstallation', function() {
...
     // switch back to system
     client.switchToFrame();
-    systemInputMgmt.switchToActiveKeyboardFrame();
+
+
+    // wait for the 2nd keyboard is loaded
+    var inputWindows = systemInputMgmt.inputWindows;
     client.waitFor(function() {
-      return imeTestApp.sendKeyButton.displayed();
+      return inputWindows.length > 1;
```
Comment (timdream) [inline apps/keyboard/test/marionette/uninstallation_test.js]: `return (inputWindows.length > 1);`

File: apps/keyboard/test/marionette/uninstallation_test.js [hunk 2]
```diff
@@ -56,10 +56,16 @@ marionette('Show Keyboard App after uninstallation', function() {
...
     });
+
+    systemInputMgmt.switchToActiveKeyboardFrame();
+    return imeTestApp.sendKeyButton.displayed();
```
Comment (timdream) [inline apps/keyboard/test/marionette/uninstallation_test.js]: Should we move the original `client.waitFor()` here or we can sure of when the input window is there the first value of `imeTestApp.sendKeyButton.displayed()` is always truthy?

Comment (RudyLu) [inline apps/keyboard/test/marionette/uninstallation_test.js]: We already use waitForElement() in imeTestApp.sendKeyButton, so I think this is fine here.
Thanks.

**Comment by try-server-hook** [comment]
[RudyLu Rudy Lu (RudyLu)](https://github.com/RudyLu) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a92b8dae282e)

**Pull request merged**